packages/zosfiles/__tests__/api/methods/create/Create.unit.test.ts

```diff
@@ -52,7 +52,42 @@ describe("Create data set", () => {
             dsOptions.dsntype = undefined;
         });
 
+        it("should be able to create an extended partitioned data set (PDSE) - test with PDS", async () => {
+
+            dsOptions.dsntype = "PDS";
+
+            const response = await Create.dataSet(dummySession, CreateDataSetTypeEnum.DATA_SET_PARTITIONED, dataSetName, dsOptions);
+
+            expect(response.success).toBe(true);
+            expect(response.commandResponse).toContain("created successfully");
+            expect(mySpy).toHaveBeenCalledWith(dummySession, endpoint, [], JSON.stringify({...CreateDefaults.DATA_SET.PARTITIONED, ...dsOptions}));
+            dsOptions.dsntype = undefined;
+        });
+
+        it("explicit testing of dsntype", async () => {
+            let success: boolean = false;
+            const dsntypes = ["BASIC", "EXTPREF", "EXTREQ", "HFS", "LARGE", "PDS", "LIBRARY", "PIPE"];
+            for (const type of dsntypes) {
+                dsOptions.dsntype = type;
+                try {
+                    await Create.dataSetValidateOptions(dsOptions);
+                } catch (err) {
+                    expect(success).toBe(true);
```

Comment:
Could we maybe output more information to see which option failed?
As it is right now, people will just see a failure on this test without much details as to what went wrong with the test itself.